packages/integration-test/test/node-client.test.ts

```diff
@@ -494,6 +495,31 @@ describe('Request client using a request node', () => {
     const requests = await badRequestNetwork.fromTopic(myRandomTopic);
     expect(requests).toHaveLength(0);
   });
+
+  it('can create multiple requests in parallel', async () => {
```

Comment:
I added this test because it seemed that the node wouldn't accespt multiple transactions at once.